integration-tests/jmx/default_instrumentation_jmx_env_variables_no_app_signals.json

```diff
@@ -0,0 +1,4 @@
+{
+  "OTEL_AWS_JMX_EXPORTER_METRICS_ENDPOINT": "http://cloudwatch-agent.amazon-cloudwatch:4314/v1/metrics",
```

Comment:
the operator should also add a couple of other environment vars like 
OTEL_EXPORTER_OTLP_PROTOCOL, OTEL_METRICS_EXPORTER and OTEL_LOGS_EXPORTER. see here - https://github.com/aws/amazon-cloudwatch-agent-operator/blob/main/pkg/instrumentation/defaultinstrumentation.go#L142